Rubberduck.Parsing/VBA/ParseCoordinator.cs

```diff
@@ -239,13 +239,10 @@ private void ExecuteCommonParseActivities(IReadOnlyCollection<QualifiedModuleNam
             _parserStateManager.SetStatusAndFireStateChanged(this, ParserState.LoadingReference, token);
             token.ThrowIfCancellationRequested();
 
-            //TODO: Remove the conditional compilation after loading from typelibs actually works.
-#if LOAD_USER_COM_PROJECTS
```

Comment:
This change is not mentioned in the PR comments and really should have been its own PR as it activates a part that can potentially cause RD to crash the host. This is also a candidate for the cause of the appearence of #5217.